plugin-enclave-gradle/src/main/kotlin/com/r3/conclave/plugin/enclave/gradle/gramine/GenerateGramineBundle.kt

```diff
@@ -82,74 +103,79 @@ open class GenerateGramineBundle @Inject constructor(
             .splitToSequence("\n")
             .single { it.startsWith("Location: ") }
             .substringAfter("Location: ")
+        val command = prepareManifestGenerationCommand(
+            architecture,
+            pythonLdPreload,
+            pythonPackagesPath,
+            manifestTemplatePath
+        )
 
-        generateManifest(architecture, ldPreload, pythonPackagesPath)
-
-        if (enclaveMode != EnclaveMode.SIMULATION) {
-            generateSgxManifestAndSigstruct()
-            generateToken()
-            // The .manifest is not needed for debug and release modes
-            outputDir.file(GRAMINE_MANIFEST).get().asFile.toPath().deleteExisting()
+        project.exec { spec ->
+            spec.commandLine = command
+            spec.setWorkingDir(outputDir)
         }
     }
 
-    private fun generateManifest(architecture: String, ldPreload: String, pythonPackagesPath: String) {
-        val manifestTemplateFile = temporaryDir.resolve(MANIFEST_TEMPLATE).toPath()
-        javaClass.copyResource(MANIFEST_TEMPLATE, manifestTemplateFile)
+    private fun generateManifestForJavaEnclaves(manifestTemplatePath: String) {
+        val command = prepareManifestGenerationCommand(
+            DOCKER_IMAGE_ARCHITECTURE,
+            "",
+            "",
+            manifestTemplatePath
+        )
+        linuxExec.exec(command, listOf("-w", outputDir.get().asFile.absolutePath))
```

Comment:
Why not just use `execCommand`? Yes, we know that it's not Python here, but it avoids some duplication and eventually Python will be executed inside the container.

Reply:
Yes, I did exactly because Python enclaves are not yet in. Fine to use it, sure!